### DISCUSSION TITLE
Path for Extbase Model

### DISCUSSION THREAD

**Discussion by TokeHerkild** [sourcebroker/t3api#32] [Q&A] [answered]
Path:
ext:Classes/Domain/Model/ 

Does it support subtrees eg:
.../Model/Reference/Kids
.../Model/Reference/Juniors
.../Model/Reference/Adults

Just having a large project with an extensive Domain tree, and would
like to know if this is supported, before I spent 3-6 hours trying to
make it work.

**Comment by klamparski** [comment, marked as answer]
Hi @TokeHerkild. Unfortunately it's not supported yet. I've already started implementation of that feature recently but couldn't finish yet as other more urgent things occurred meanwhile. Sorry but it's hard to say when I will be able to finish it.

**Reply by TokeHerkild** [reply]
Thanks for the answer. to bad though ;)

**Reply by klamparski** [reply]
@TokeHerkild You inspired me to finally move forward this feature. It already works but not yet merged to master because I need to do some functional tests and documentation. Feature will be included in next release.
You can already use it by switching to branch `issue-22`. Try to install t3api using `composer require sourcebroker/t3api:dev-issue-22` and please let me know if it worked as expected for you.

btw. here is the issue for that feature: https://github.com/sourcebroker/t3api/issues/22
